### PR TITLE
Updating Excel device info for DynamicImageDpi

### PR DESCRIPTION
**Pull request by onegoodsausage**
Updating Excel device info settings for the addition of new DynamicImageDpi device info.

### CODE REVIEW

**Comment by PRMerger17** [comment]
@onegoodsausage : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/reporting-services/excel-device-information-settings.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ ms.author: maggies
 |-------------|-----------|  
 |**OmitDocumentMap**|Indicates whether to omit the document map for reports that support it. The default value is **false**.|  
 |**OmitFormulas**|Indicates whether to omit formulas from the rendered report. The default value is **false**.|  
-|**SimplePageHeade**rs|Indicates whether the page header of the report is rendered to the Excel page header. A value of **false** indicates that the page header is rendered to the first row of the worksheet. The default value is **false**.|  
+|**SimplePageHeaders**|Indicates whether the page header of the report is rendered to the Excel page header. A value of **false** indicates that the page header is rendered to the first row of the worksheet. The default value is **false**.|  
+|**DynamicImageDpi**|The resolution of dynamic images like Charts, Gauges and Maps. The default value is **96**. [Available in Power BI Report Server (January 2020) and later]|
```
Comment (maggiesMSFT) [inline docs/reporting-services/excel-device-information-settings.md]:
```suggestion
|**DynamicImageDpi**|The resolution of dynamic images like charts, gauges and maps. The default value is **96**. (Available in Power BI Report Server (January 2020) and later)|  

```

**Comment by maggiesMSFT** [comment]
#sign-off

**Comment by PRMerger17** [comment]
@onegoodsausage : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by maggiesMSFT** [comment]
#sign-off

**Pull request merged**